src/riscv_private.h

```diff
@@ -119,8 +126,12 @@ struct riscv_internal {
     uint32_t csr_mip;
     uint32_t csr_mbadaddr;
 
-    bool compressed;       /**< current instruction is compressed or not */
+    bool compressed; /**< current instruction is compressed or not */
+#if !RV32_HAS(JIT)
     block_map_t block_map; /**< basic block map */
+#else
+    struct cache *cache;
```

Comment:
To minimize the code changes, is it possible to stick to `block_map_t` rather than `struct cache`?

Reply:
It is difficult to stick to `block_map_t`, because the structure of ARC cache is totally different from `block_map_t`.